### PR TITLE
show random username

### LINKED ISSUE #57: random username assignment
Assign random username to users when they join for chat

### PR DESCRIPTION
**Pull request by jainkeval**
fixes #57  

https://aqueous-mesa-78178.herokuapp.com/chat

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +19,9 @@ app.get('/', (req, res) => {
 });
 
 app.get('/chat', (req, res) => {
-  res.render('chat');
+  var randomName = faker.name.findName();
```
Comment (sumedh123) [inline index.js]: es6 syntax.

File: index.js [hunk 2]
```diff
@@ -16,7 +19,9 @@ app.get('/', (req, res) => {
...
+  console.log(randomName);
```
Comment (sumedh123) [inline index.js]: Please dont keep logging statements when you make a pr.

File: index.js [hunk 3]
```diff
@@ -16,7 +19,9 @@ app.get('/', (req, res) => {
...
+  res.render('chat', {userName: randomName});
```
Comment (sumedh123) [inline index.js]: Please follow the style guide. space after curly, etc.

Comment (jainkeval) [reply index.js]: ohkk..sorry for that.

**Pull request merged**